index.bs

```diff
@@ -11126,13 +11126,18 @@ and is described in more detail in [[#interface-prototype-object]].
 Note: Since an [=interface object=] is a [=function object=]
 the <code>typeof</code> operator will return "function" when applied to an interface object.
 
+An interface may have <dfn export>overridden constructor steps</dfn>, which can
+change the behavior of the [=interface object=] when called or constructed. By
+default interfaces do not have such steps.
+
 <div algorithm>
 
     The [=interface object=] for a given [=interface=] |I|
     with [=identifier=] |id| and in [=Realm=] |realm|
     is <dfn lt="create an interface object">created</dfn> as follows:
 
-    1.  Let |steps| be the following steps:
+    1.  Let |steps| be |I|'s [=overriden constructor steps=] if they exist, or
+        the following steps otherwise:
         1.  If |I| was not declared with a [=constructor operation=],
             then [=ECMAScript/throw=] a {{ECMAScript/TypeError}}.
         1.  If {{NewTarget}} is <emu-val>undefined</emu-val>, then
```

Comment:
Should we keep the first two steps, even when overriding the rest of the algorithm?

Maybe also the last four steps?

Reply:
I guess that would work for [HTMLConstructor], although it feels cleaner to just have a general primitive that replaces the whole thing.

Reply:
I'll keep it like this for now, but we can refactor later if you think it's much better.